### PR TITLE
Created an empty windows form for dlgUseTable.

### PR DESCRIPTION
**Pull request by anastasia-mbithe**
@N-thony and @lloyddewit 

I have added an empty windows form for dlgUseTable, in line with #8081.

### CODE REVIEW

File: instat/frmMain.Designer.vb [hunk 1]
```diff
@@ -5094,6 +5095,13 @@ Partial Class frmMain
         Me.mnuLogFile.Text = "Log Window..."
         Me.mnuLogFile.ToolTipText = "Log Window"
         '
+        'mnuDescribeUseTable
+        '
+        Me.mnuDescribeUseTable.Enabled = False
```
Comment (lloyddewit) [inline instat/frmMain.Designer.vb:5100]: please could you make visible also false?

File: instat/frmMain.vb [hunk 2]
```diff
@@ -1416,27 +1416,27 @@ Public Class frmMain
...
         ctrActive = ucrLogWindow
     End Sub
 
-    Private Sub ucrColumnMeta_Enter(sender As Object, e As EventArgs) Handles ucrColumnMeta.Enter
+    Private Sub ucrColumnMeta_Enter(sender As Object, e As EventArgs)
         ctrActive = ucrColumnMeta
     End Sub
 
-    Private Sub ucrDataFrameMeta_Enter(sender As Object, e As EventArgs) Handles ucrDataFrameMeta.Enter
+    Private Sub ucrDataFrameMeta_Enter(sender As Object, e As EventArgs)
```
Comment (lloyddewit) [inline instat/frmMain.vb:1439]: was it the intention to remove these events?
if so, then should we also delete these subs?

**Review by lloyddewit — CHANGES_REQUESTED** [review]
Thanks, just a couple of questions.
Note: @MeSophie is also changing `frmMain.Designer.vb'. Whoever merges first will cause a merge conflict for the other. You may want to coordinate with her.

File: instat/frmMain.vb (hunk 2 above)
Comment (anastasia-mbithe) [reply instat/frmMain.vb]: Nop, Its just something weird I've noticed. I didnt delete the events, it did so after pushing the branch. But I've fixed that now.

File: instat/frmMain.Designer.vb (hunk 1 above)
Comment (anastasia-mbithe) [reply instat/frmMain.Designer.vb]: done now

File: instat/frmMain.Designer.vb [hunk 3]
```diff
@@ -5107,9 +5116,7 @@ Partial Class frmMain
         Me.IsMdiContainer = True
         Me.MainMenuStrip = Me.mnuBar
         Me.Name = "frmMain"
-        Me.Text = "R-Instat  " + My.Application.Info.Version.Major.ToString + "." +
-                My.Application.Info.Version.Minor.ToString + "." +
-                My.Application.Info.Version.Build.ToString
+        Me.Text = "R-Instat  .."
```
Comment (lloyddewit) [inline instat/frmMain.Designer.vb:5119]: I think that this change is a side effect of opening frmMain in the designer.
Please could you manually reverse this change?
thanks

**Review by lloyddewit — CHANGES_REQUESTED** [review]

**Comment by anastasia-mbithe** [comment]
@lloyddewit I made the changes

**Review by N-thony — APPROVED** [review]

**Comment by anastasia-mbithe** [comment]
> Thanks, just a couple of questions. Note: @MeSophie is also changing `frmMain.Designer.vb'. Whoever merges first will cause a merge conflict for the other. You may want to coordinate with her.

@lloyddewit you can merge hers first, I can redo my PR if I won't be able to solve the conflicts.

**Review by lloyddewit — APPROVED** [review]

**Pull request merged**